seasmon_xr/accessors.py

```diff
@@ -567,3 +568,77 @@ def autocorr(self):
             dask="parallelized",
             output_dtypes=["float32"],
         )
+
+
+@xarray.register_dataarray_accessor("zonal")
```

Comment:
Currenly this is only tied to dataarray - not sure if we want to extend this to dataset